tox.ini

```diff
@@ -82,5 +82,5 @@ deps=
   teamcity-messages
 changedir=packages/bootstrap/extra
 commands=
-  python setup.py install
+  pip install .
```

Comment:
Should actually replace all `pip` usage in our codebase with `pip3` (This is far from the only one)